site/src/components/layout.scss

```diff
@@ -1,6 +1,10 @@
 @charset "utf-8";
-@import "~hds-design-tokens/lib/all.scss";
+@import "~hds-core/lib/base.min.css";
 @import "~hds-core/lib/components/all.min.css";
+@import "~hds-core/lib/components/table/table.css";
```

Comment:
`table.css` isn't needed because the styles are included in `@import "~hds-core/lib/components/all.min.css";`

Reply:
Ok I'll remove it 👍